Compiler-Warnings in MeshingApplication

Some compilation warnings:
~~~
/home/philippb/software/Kratos_prod/applications/MeshingApplication/tests/cpp_tests/test_uniform_refine_utility.cpp: In member function ‘virtual void Kratos::Testing::TestUniformRefineQuadrilateralsUtility::TestFunction()’:
/home/philippb/software/Kratos_prod/applications/MeshingApplication/tests/cpp_tests/test_uniform_refine_utility.cpp:187:24: warning: unused variable ‘r_sub_model_part_2’ [-Wunused-variable]
             ModelPart& r_sub_model_part_2 = this_model_part.CreateSubModelPart("SkinSubModelPart");
~~~

and 

~~~
/home/philippb/software/Kratos_prod/applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp: In function ‘<built-in>’:
/home/philippb/software/Kratos_prod/applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp:107:103: warning: ‘element_max_size’ may be used uninitialized in this function [-Wmaybe-uninitialized]
         const double element_max_size = ((element_max_size > nodal_h) && mEnforceCurrent) ? nodal_h : mMaxSize;
                                                                                                       ^
/home/philippb/software/Kratos_prod/applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp:107:22: note: ‘element_max_size’ was declared here
         const double element_max_size = ((element_max_size > nodal_h) && mEnforceCurrent) ? nodal_h : mMaxSize;
                      ^
/home/philippb/software/Kratos_prod/applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp:106:103: warning: ‘element_min_size’ may be used uninitialized in this function [-Wmaybe-uninitialized]
         const double element_min_size = ((element_min_size > nodal_h) && mEnforceCurrent) ? nodal_h : mMinSize;
                                                                                                       ^
/home/philippb/software/Kratos_prod/applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp:106:22: note: ‘element_min_size’ was declared here
         const double element_min_size = ((element_min_size > nodal_h) && mEnforceCurrent) ? nodal_h : mMinSize;
~~~

--- a/applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp
+++ b/applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp
@@ -103,8 +103,10 @@ void ComputeHessianSolMetricProcess<TDim, TVarType>::Execute()
         KRATOS_DEBUG_ERROR_IF_NOT(it_node->SolutionStepsDataHas(NODAL_H)) << "ERROR:: NODAL_H not defined for node " << it_node->Id();
         const double nodal_h = it_node->FastGetSolutionStepValue(NODAL_H);            
         
-        const double element_min_size = ((element_min_size > nodal_h) && mEnforceCurrent) ? nodal_h : mMinSize;
-        const double element_max_size = ((element_max_size > nodal_h) && mEnforceCurrent) ? nodal_h : mMaxSize;
+        double element_min_size = mMinSize;
+        if ((element_min_size > nodal_h) && mEnforceCurrent) element_min_size = nodal_h;
+        double element_max_size = mMaxSize;
+        if ((element_max_size > nodal_h) && mEnforceCurrent) element_max_size = nodal_h;
 
         // Isotropic by default
         double ratio = 1.0;

--- a/applications/MeshingApplication/tests/cpp_tests/test_uniform_refine_utility.cpp
+++ b/applications/MeshingApplication/tests/cpp_tests/test_uniform_refine_utility.cpp
@@ -184,7 +184,7 @@ namespace Kratos
 
             // Creating the sub model parts
             ModelPart& r_sub_model_part_1 = this_model_part.CreateSubModelPart("BodySubModelPart");
-            ModelPart& r_sub_model_part_2 = this_model_part.CreateSubModelPart("SkinSubModelPart");
+//             ModelPart& r_sub_model_part_2 = this_model_part.CreateSubModelPart("SkinSubModelPart");
 
             // Creating the nodes
             NodeType::Pointer p_node_1 = this_model_part.CreateNewNode(1, 0.0, 0.0, 0.0);

